app/main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.learn_skill("JavaScript")
+        self.learn_skill("HTML")
+        self.learn_skill("CSS")
```

Comment:
To simplify the code and add all the skills at once instead of calling the method multiple times, you can use the self.skills += [] or self.skills.extend([]) approach. This allows you to add multiple skills to the self.skills list in a single operation. See checklist
![checklist](https://github.com/mate-academy/py-software-engineers/assets/98127979/1bfb6610-91d4-47f7-a258-fe8f4c4df9fb)